src/service/user-api-adapter.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+/**
+* ##User API Adapter
+*
+* The user API adapter allows you to quert the /user API
+*
+*
+*/
+
+var ConfigService = require('./configuration-service');
+var TransportFactory = require('../transport/http-transport-factory');
+var qutil = require('../util/query-util');
+
+module.exports = function (config) {
+    var defaults = {
+
+       account: '',
+
+       token: '',
+
+        //Options to pass on to the underlying transport layer
+        transport: {}
+    };
+
+    var serviceOptions = $.extend({}, defaults, config);
+    var urlConfig = new ConfigService(serviceOptions).get('server');
+    var transportOptions = $.extend(true, {}, serviceOptions.transport, {
+        url: urlConfig.getAPIPath('user')
+    });
+
+    if (serviceOptions.token) {
+        transportOptions.headers = {
+            'Authorization': 'Bearer ' + serviceOptions.token
+        };
+    }
+
+    if (serviceOptions.apiKey) {
```

Comment:
Can we specify both token and apiKey? Also, is the apiKey valid for other services also?